Apps/ValidationTests/Scripts/validation_native.js

```diff
@@ -51,15 +51,20 @@ function compare(test, renderData, referenceImage, threshold, errorRatio) {
     return error;
 }
 
-function evaluate(test, resultCanvas, result, referenceImage, index, waitRing, done) {
+function saveRenderedResult(test, renderData) {
+    TestUtils.writePNG(renderData, testWidth, testHeight, TestUtils.getOutputDirectory() + "/Results/" + test.referenceImage);
```

Comment:
I noticed this function writes out uncompressed PNG files. Are we okay with that? Especially since they are merged in the repo.

Reply:
Not ideal, indeed. A potential improvement is adding a more advanced png writer in validationtests app.

Reply:
Might be something worth doing as to not bloat the repo with big pngs.

Reply:
I take note.